### DISCUSSION TITLE
RainbowMiner mines from pools that I haven't enabled

### DISCUSSION THREAD

**Discussion by stevegwh** [RainbowMiner/RainbowMiner#1678] [Q&A] [answered]
When starting the program the output lists the mining pools that I have configured. However, normally after some time, the miner switches to mining in pools (and output currencies) that I have not configured. For example, I have excluded Nicehash in the configuration files yet it switches to mining Nicehash. 

How can I stop this from happening? I have run through the setup multiple times and confirmed my list of pools in the configuration files.

Platform: Windows
Devices: AMD/CPU
RainbowMiner version: 4.7.2.1

**Comment by RainbowMiner** [comment, marked as answer]
There are two possibilities:
- RainbowMiner mines to my pools for some minutes per day to support my work a bit. It's the donation round, the length can be controlled via the parameter "Donate" in config.txt.
- if you have setup RainbowMiner for MiningRigRentals, some algorithms cannot be covered with pools like ZergPool, Nicehash or Zpool. To be able to benchmark all possible MRR algorithms, RainbowMiner will benchmark to some of my pools (only, if those algos aren't covered with your setup, already).

In your case, it's most probably the donation round.

**Comment by stevegwh** [comment]
> There are two possibilities:
> 
> * RainbowMiner mines to my pools for some minutes per day to support my work a bit. It's the donation round, the length can be controlled via the parameter "Donate" in config.txt.
> * if you have setup RainbowMiner for MiningRigRentals, some algorithms cannot be covered with pools like ZergPool, Nicehash or Zpool. To be able to benchmark all possible MRR algorithms, RainbowMiner will benchmark to some of my pools (only, if those algos aren't covered with your setup, already).
> 
> In your case, it's most probably the donation round.

Ah, I thought this might be the case :). Thanks for clearing that up!
